component/common/appendable/appendable.go

```diff
@@ -16,13 +16,15 @@ type FlowMetric struct {
 	Value  float64
 }
 
-type flowAppendable struct {
-	receivers []*metrics.Receiver
+// FlowAppendable is a flow-specific implementation of an Appender.
+type FlowAppendable struct {
+	Receivers []*metrics.Receiver
 }
```

Comment:
nit: why not `type FlowAppendable []*metrics.Receiver`?

Reply:
Hmm, this used to contain more fields when we hadn't separated the `FlowAppendable` and the per-scrape job  derived `appender`.

I'll change it to this one-liner type alias, and we can return to the struct definition if we ever need more fields again.